packages/base/command.gts

```diff
@@ -49,3 +51,41 @@ export class CommandCard extends CardDef {
   @field eventId = contains(StringField);
   @field status = contains(CommandStatusField);
 }
+
+export class SaveCardInput extends CardDef {
+  @field realm = contains(StringField);
+  @field card = linksTo(CardDef);
+}
+
+class JsonField extends FieldDef {
+  static [primitive]: Record<string, any>;
+}
+
+export class PatchCardInput extends CardDef {
+  @field cardId = contains(StringField);
+  @field patch = contains(JsonField); //TODO: JSONField ?
+}
+
+export class ShowCardInput extends CardDef {
+  @field cardToShow = linksTo(CardDef);
+  @field placement = contains(StringField); // TODO: nicer if enum, likely need to specify stackIndex too?
```

Comment:
take a look at base64-image for an example of enum field

Reply:
Thanks! Removing this field for now. Will revisit